<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matrix Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pageMatrixInput">
     <widget class="QLabel" name="m1r_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Rows</string>
      </property>
     </widget>
     <widget class="QLabel" name="m1_label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Matrix 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="m1c_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cols</string>
      </property>
     </widget>
     <widget class="QLabel" name="m1e_label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Elements</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="r1_sb">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="c1_sb">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QTableWidget" name="m1_tab">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>371</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="m1e_in">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="m2c_label">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cols</string>
      </property>
     </widget>
     <widget class="QLabel" name="m2_label">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Matrix 2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m2e_in">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>160</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="m2e_label">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>160</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Elements</string>
      </property>
     </widget>
     <widget class="QLabel" name="m2r_label">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>100</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Rows</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="c2_sb">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>130</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="r2_sb">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QTableWidget" name="m2_tab">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>200</y>
        <width>371</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="next_btn">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>20</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Next →</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageOperations">
     <widget class="QPushButton" name="back_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>← Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="m1_label_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Matrix 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="m2_label_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Matrix 2</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="res_tab">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>371</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Addition</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sub_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>130</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Subtraction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mul_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>160</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Multiplication</string>
      </property>
     </widget>
     <widget class="QPushButton" name="det1_btn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Determinant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="det2_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Determinant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="t1_btn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Transpose</string>
      </property>
     </widget>
     <widget class="QPushButton" name="t2_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>130</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Transpose</string>
      </property>
     </widget>
     <widget class="QPushButton" name="inv1_btn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>160</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Inverse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="inv2_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>160</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Inverse</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMatric_Calculator_Group_4">
    <property name="font">
     <font>
      <family>Fira Code</family>
     </font>
    </property>
    <property name="title">
     <string>Matrix Calculator - Group 4</string>
    </property>
   </widget>
   <addaction name="menuMatric_Calculator_Group_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
